iati/about/models.py

```diff
@@ -1,8 +1,145 @@
 from django.db import models
 
-from wagtail.core.models import Page
+from wagtail.admin.edit_handlers import InlinePanel
+from wagtail.core.blocks import CharBlock, StreamBlock, StructBlock, TextBlock
+from wagtail.core.fields import StreamField
+from wagtail.core.models import Orderable
+from wagtail.documents.edit_handlers import DocumentChooserPanel
+from wagtail.images.blocks import ImageChooserBlock
+from wagtail.images.edit_handlers import ImageChooserPanel
 
-class AboutPage(Page):
-    pass
-    # parent_page_types = ['home.HomePage']
-    # subpage_types = ['home.AboutSubpage']
+from modelcluster.fields import ParentalKey
+from home.models import AbstractContentPage, AbstractIndexPage, IATIStreamBlock
+
+
+class AboutPage(AbstractContentPage):
+    """A model for the About landing page."""
+
+    parent_page_types = ['home.HomePage']
+    subpage_types = ['about.AboutSubPage', 'about.CaseStudyIndexPage', 'about.HistoryPage', 'about.PeoplePage']
+
+
+class AboutSubPage(AbstractContentPage):
+    """A model for generic About subpages."""
+
+    subpage_types = ['about.AboutSubPage']
+
+    multilingual_field_panels = [
+        InlinePanel('about_sub_page_documents', label='About subpage attachments'),
+    ]
+
+
+class AboutSubPageDocument(Orderable):
+    """A model for About sub page documents."""
+
+    page = ParentalKey(AboutSubPage, related_name='about_sub_page_documents')
+    document = models.ForeignKey(
+        'wagtaildocs.Document',
+        on_delete=models.CASCADE,
+        related_name='+'
+    )
+
+    panels = [
+        DocumentChooserPanel('document'),
+    ]
+
+
+class CaseStudyIndexPage(AbstractIndexPage):
+    """A model for the Case Studies Index page."""
+
+    subpage_types = ['about.CaseStudyPage']
+
+    @property
+    def case_studies(self):
+        """Get all CaseStudyPage objects that have been published."""
```

Comment:
Not something for now, but this is a good example (to me) of where it would be useful to show the return object type in the docstring.

According to @akmiller01 this will be a `QuerySet`